src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -68,7 +67,94 @@ <h4 class="panel-title border-bottom-small" translate>
         <h4 class="panel-title border-bottom-small" translate>
           {{'GIFT_AMOUNT'}}
         </h4>
-        <div class="panel-body pt0">
+        <div class="panel-body p0" ng-if="$ctrl.useV3 == 'true'">
+          <fieldset aria-label="Suggested Gift Amounts. Tab for Custom Amount">
```

Comment:
I think this label and fieldset need to be within the conditional (probably just move the `ng-if` up to this element)

Reply:
You're referring to the <h4>?

Reply:
No, I'm referring to the `fieldset`. It should not be in the DOM if there are no suggested amounts.

Reply:
Oh I see. If I'm understanding the use of ng-if, it won't be in the DOM if the ng-if is in the parent `<div>` above it right?